java-persistence/src/main/java/com/whitecloak/training/inventory/model/Item.java

```diff
@@ -1,10 +1,15 @@
 package com.whitecloak.training.inventory.model;
 
+import com.whitecloak.training.inventory.persistence.entity.CategoryEntity;
+import com.whitecloak.training.inventory.persistence.entity.UserEntity;
+
 public class Item {
 
     private Long id;
     private String name;
-    private Long ownerId;
+    private UserEntity ownerId;
+    private CategoryEntity categoryId;
```

Comment:
Relationships does not need to be named `*Id` -- `owner` and `category` will do. The data types, `UserEntity` and `CategoryEntity` are correct.